src/js/common/schemaform/state/reducers.js

```diff
@@ -12,6 +15,22 @@ import {
 } from '../state/helpers';
 
 export default {
+  [OPEN_REVIEW_CHAPTER]: (state, action) => {
+    const openChapters = [
+      ...state.reviewPageView.openChapters,
+      action.openedChapter
+    ];
+
+    return _.set('reviewPageView.openChapters', openChapters, state);
+  },
+  [CLOSE_REVIEW_CHAPTER]: (state, action) => {
```

Comment:
This is very satisfying to see.
![redux](https://user-images.githubusercontent.com/4998130/38595443-174e9002-3d09-11e8-8951-c2967749bf90.gif)